python/tk_multi_workfiles/ui/file_save_form.py

```diff
@@ -166,7 +166,7 @@ def setupUi(self, FileSaveForm):
         self.verticalLayout_4.setObjectName("verticalLayout_4")
         self.preview_grid = QtGui.QGridLayout()
         self.preview_grid.setHorizontalSpacing(14)
-        self.preview_grid.setVerticalSpacing(6)
+        self.preview_grid.setVerticalSpacing(1)
```

Comment:
I'm guessing this is a result of modifying the UI file and recompiling but isn't an intentional change. Does it make any visual difference?

Reply:
There was an issue with the warning message being clipped now that it is longer. This seems to solve the problem. I'd love to come up with a better solution but I couldn't figure out the root of the problem